scipy-1.0/paper.tex

```diff
@@ -942,10 +943,13 @@ \subsubsection*{Test suite}
 The SciPy test suite is orchestrated by a continuous integration matrix that
 includes POSIX and Windows (32/64-bit) platforms managed by Travis CI and
 AppVeyor, respectively. Our tests cover Python versions 2.7, 3.4, 3.5, 3.6, and
-include code linting with \texttt{pyflakes} and \texttt{pycodestyle}. There are more than $13,000$
-unit tests in the test suite, which is written for usage with the \texttt{pytest}
-framework, and with 87\% and 45\% line coverages for Python and compiled
-code, respectively at the SciPy 1.0 release point (Figure~\ref{fig:coverage}). Documentation for the code is automatically built and published by
+include code linting with \texttt{pyflakes} and \texttt{pycodestyle}. There
```

Comment:
Added citations for things below. Probably should have added citations for these.

Reply:
We've been *extremely* comprehensive in citing both papers and online tools, so maybe we can cut ourselves a little slack here as we head down the stack to linters, but fine by me if you want to cite those too :)